demo/api.md

```diff
@@ -46,12 +46,12 @@
 
 ## CSS Shadow Parts
 
-| Part          | Description                |
-|---------------|----------------------------|
-| [button](#button)      | Apply CSS to HTML5 button. |
-| `button-icon` | Apply CSS to icon slot.    |
-| `button-text` | Apply CSS to text slot.    |
-| [loader](#loader)      | Apply CSS to auro-loader.  |
+| Part     | Description                |
+|----------|----------------------------|
+| [button](#button) | Apply CSS to HTML5 button. |
+| [icon](#icon)   | Apply CSS to icon slot.    |
+| [loader](#loader) | Apply CSS to auro-loader.  |
+| [text](#text)   | Apply CSS to text slot.    |
```

Comment:
Changing the part names would make this a breaking change. 

This change isn't justified in the linked tickets or the PR description.

Reply:
Turns out this was just a documentation issue. Previous release did not rebuild docs.